metadata.yaml

```diff
@@ -0,0 +1,13 @@
+# maps release series of major.minor to cluster-api contract version
+# the contract version may change between minor or major versions, but *not*
+# between patch versions.
+#
+# update this file only when a new major or minor version is released
+apiVersion: clusterctl.cluster.x-k8s.io/v1alpha3
```

Comment:
/assign @wfernandes to validate this is what we want

Reply:
lgtm. 
For reference,
https://cluster-api.sigs.k8s.io/clusterctl/provider-contract.html#metadata-yaml